nrt/data/IEEE14/MC_file_div/reference/aggregatedResults.xml

```diff
@@ -11,15 +11,15 @@
     <scenarioResults id="DisconnectGroup" status="CONVERGENCE"/>
   </loadIncreaseResults>
   <loadIncreaseResults loadLevel="7" status="CONVERGENCE">
-    <scenarioResults id="DisconnectLine" status="CONVERGENCE"/>
```

Comment:
This nrt might need a double check as the results are different now and I don't really know what might cause it

Reply:
It is a known issue, @marcochiaramello is working on it

Reply:
Ok, so we close this MR then ?

Reply:
I think we can keep the modifications in the criteria and revert this one